### ISSUE TITLE
[BUG] - Docker container fails with default IPC socket location

### ISSUE THREAD

**Issue by tdiesler** [input-output-hk/cardano-node#2340]
**Internal/External**
*External*

**Area**
*Docker*    

**Summary**

The instructions documented [here](https://hub.docker.com/r/inputoutput/cardano-node) do not work when the IPC socket location is not mapped to a volume.

**Steps to reproduce**

This works ...

```
docker run --rm \
  -v ipc:/ipc \
  -e NETWORK=mainnet \
  inputoutput/cardano-node
```

This fails ...

```
docker run --rm \
  -e NETWORK=mainnet \
  inputoutput/cardano-node

cardano-node: Network.Socket.bind: does not exist (No such file or directory)
```

**System info (please complete the following information):**
- OS: Debian
- Version: 10
- Node version: 1.25.1

**Additional context**

You can leave it to me, I'll provide a fix.

**Comment by tdiesler** [comment]
PR: #2341

**Comment by cgi1** [comment]
While the PR is closed, this issue still seems to be valid. At least I still encounter 

`cardano-node: Network.Socket.bind: does not exist (No such file or directory)`

when using the latest docker tag.

**Comment by tdiesler** [comment]
Yes, there are various kinks with the upstream images, which are waiting to get fixed as part of this PR #2456
In the meantime, you can head over to [nessusio/cardano-node](https://hub.docker.com/repository/docker/nessusio/cardano-node) which is the incubation space to what I propose in #2360 

Comprehensive documentation is [here](https://tdiesler.gitbook.io/cardano/v/iohk/).

**Comment by tdiesler** [comment]
Merged
